### PR TITLE
Content layout

### PR DESCRIPTION
**Pull request by lucasnestrojil**
## Summary
- Setting of the content layout
// Comment: I try to set the size of the image only with the height property, but the image anchor its biggest than the container.

## Screenshots
image

## Trello Card
https://trello.com/c/P6EqBIiP/23-content-layout

### CODE REVIEW

File: maquetado/setup/index.html [hunk 1]
```diff
@@ -8,12 +8,65 @@
...
+      </div>
+      <div class="tech-front-container">
+        <img class="tech-img" alt="Polymer Logo" src="img/polymer_icon.svg" />
+        <h4 class="tech-title">Polymer</h1>
+        <h5 class="tech-release">Release: 2015</h2>
+        <p class="tech-description">Polymer is an open-source JavaScript library for building web applications using Web Components.</p>
+      </div>
+      </div>
+      <h2 class="title">Back-End Tools</h2>  
+      <div class="tech-catalog">
```
Comment (mmamaniwolox) [inline maquetado/setup/index.html]: Also this

File: maquetado/setup/index.html [hunk 2]
```diff
@@ -8,12 +8,65 @@
...
+        <h5 class="tech-release">Release: 2016</h2>
+        <p class="tech-description">Elm is a domain-specific programming language for declaratively creating web browser-based graphical user interfaces.</p>
+      </div>
+      <div class="tech-front-container">
+        <img class="tech-img" alt="Polymer Logo" src="img/polymer_icon.svg" />
+        <h4 class="tech-title">Polymer</h1>
+        <h5 class="tech-release">Release: 2015</h2>
+        <p class="tech-description">Polymer is an open-source JavaScript library for building web applications using Web Components.</p>
+      </div>
+      </div>
```
Comment (mmamaniwolox) [inline maquetado/setup/index.html]: This has two extra spaces

File: maquetado/setup/index.html [hunk 3]
```diff
@@ -8,12 +8,65 @@
...
+      <h2 class="title">Back-End Tools</h2>
```
Comment (mmamaniwolox) [inline maquetado/setup/index.html]: this too

File: maquetado/setup/index.html [hunk 4]
```diff
@@ -8,12 +8,65 @@
...
+      </div>
+      <h2 class="title">Back-End Tools</h2>  
+      <div class="tech-catalog">
+      <div class="tech-front-container">
+        <img class="tech-img" alt="Ruby on Rails Logo" src="img/rails_icon.png" />
+        <h4 class="tech-title">Ruby on Rails</h1>
+        <h5 class="tech-release">Release: 2005</h2>
+        <p class="tech-description">Ruby on Rails, or Rails, is a server-side web application framework written in Ruby. Rails is an MVC framework, providing default structures for a database, a web service, and web pages.</p>
+      </div>
+      <div class="tech-front-container">
```
Comment (mmamaniwolox) [inline maquetado/setup/index.html]: This techs are not `front`, what do you think of changing the class name for one more descriptive?

File: maquetado/setup/index.html [hunk 5]
```diff
@@ -8,12 +8,65 @@
...
-      <h5 class="tech-release">Release: 2013</h2>
-      <p class="tech-description">React is a declarative, component-based JavaScript library for building user interfaces.</p>
-    </div>  
+    <h2 class="title">Front-End Tools</h2>    
+    <div class="tech-catalog">
+      <div class="tech-front-container">
+        <img class="tech-img" alt="React Logo" src="img/react_icon.svg" />
+        <h4 class="tech-title">React</h1>
+        <h5 class="tech-release">Release: 2013</h2>
+        <p class="tech-description">React is a declarative, component-based JavaScript library for building user interfaces.</p>
```
Comment (mmamaniwolox) [inline maquetado/setup/index.html]: You can try splitting 


```
<p class="tech-description">
  React is a declarative, component-based JavaScript library for building user interfaces.
</p>
```

File: maquetado/setup/scss/tech.scss [hunk 6]
```diff
@@ -18,13 +18,14 @@
...
-    width: 190px;  
-    height: 250px;
+    width: 210px;  
+    height: 300px;
   }
 }
 
 .tech-img {
   height: 100px;
+  width: 100px;
```
Comment (mmamaniwolox) [inline maquetado/setup/scss/tech.scss:28]: If the width is your limitting att, try removing the height. If it doesnt show the way you want that way, you can dismiss this comment

**Review by mmamaniwolox — CHANGES_REQUESTED** [review]

File: maquetado/setup/scss/tech.scss (hunk 6 above)
Comment (lucasnestrojil) [reply maquetado/setup/scss/tech.scss:28]: My limiting attributes are both, therefore i used them

File: maquetado/setup/index.html [hunk 7]
```diff
@@ -8,14 +8,86 @@
...
-      </p>
-    </div>  
+    <h2 class="title">Front-End Tools</h2>    
+    <div class="tech-catalog">
+      <div class="tech-container">
+        <img class="tech-img" alt="React Logo" src="img/react_icon.svg" />
+        <h4 class="tech-title">React</h1>
+        <h5 class="tech-release">Release: 2013</h2>
+        <p class="tech-description">
+        React is a declarative, component-based JavaScript library for building user interfaces.
```
Comment (mmamaniwolox) [inline maquetado/setup/index.html]: Add two more spaces in all this text cases

**Review by mmamaniwolox — CHANGES_REQUESTED** [review]

File: maquetado/setup/scss/titles.scss [hunk 8]
```diff
@@ -1,8 +1,10 @@
 .title {
-  display: flex;
   align-items: center;
+  display: flex;
   font-size: 24px;
-  margin: 50px 0px 0px 100px;
+  margin: 50px 0px 50px 0px;
```
Comment (mmamaniwolox) [inline maquetado/setup/scss/titles.scss:5]: You can replace this for `margin: 50px 0`

**Review by mmamaniwolox — APPROVED** [review]

**Pull request merged**